src/app/templates/index.ftl

```diff
@@ -1,27 +1,42 @@
 <#-- @ftlvariable name="reportsJson" type="String" -->
 <#-- @ftlvariable name="isReviewer" type="Boolean" -->
+<#-- @ftlvariable name="isReportRunner" type="Boolean" -->
 <#-- @ftlvariable name="showProjectDocs" type="Boolean" -->
 <#macro if if then else=""><#if if>${then}<#else>${else}</#if></#macro>
 <@layout>
     <#macro styles>
         <link rel="stylesheet" href="${appUrl}/css/index.min.css"/>
     </#macro>
     <#include "partials/pinned-reports.ftl">
-    <#if showProjectDocs>
-        <h3 class="mb-3">
+
+    <h3 class="mb-3">
+        <#if showProjectDocs>
             <button class="btn-link btn btn-lg"><a href="${appUrl}/project-docs">View project documentation
-                <svg height="40" class="octicon octicon-arrow-right"
-                     viewBox="0 0 10 16"
-                     version="1.1"
-                     width="30"
-                     aria-hidden="true"
-                     fill="currentColor">
-                    <path fill-rule="evenodd" d="M10 8L4 3v3H0v4h4v3l6-5z"></path>
-                </svg>
+                    <svg height="40" class="octicon octicon-arrow-right"
+                         viewBox="0 0 10 16"
+                         version="1.1"
+                         width="30"
+                         aria-hidden="true"
+                         fill="currentColor">
+                        <path fill-rule="evenodd" d="M10 8L4 3v3H0v4h4v3l6-5z"></path>
+                    </svg>
```

Comment:
You could define this as a [macro](https://freemarker.apache.org/docs/ref_directive_macro.html) to avoid repetition